codice/ddf
# [2.11.x] DDF-3604 Fixed OpenSearch bounding box

#### What does this PR do?
This PR fixes the OpenSearch `Convert to BBox` option.
#### Who is reviewing it? 
@AzGoalie @peterhuffer 
#### Select relevant component teams: 
@codice/io 
#### Choose 2 committers to review/merge the PR. 
@bdeining
@clockard
#### How should this be tested?
Execute point radius searches to an OpenSearch source in Intrigue, and confirm that the expected results are returned.
#### Any background context you want to provide?
Specifically, the issue was observed with a record with `location`=`POLYGON ((-122 37.33, -122.05 37.33, -122.05 37.22, -122 37.22, -122 37.33))` and the search "`Latitude`=`33`, `Longitude`=`-122`, `Radius`=`500 miles`" from Intrigue. The `OpenSearchSource`, with `Convert to BBox`=`true`, was incorrectly transforming the spatial search to `bbox=422.42053621382615,25.77186615764653,-666.4205362138262,40.22813384235347`.
#### What are the relevant tickets?
[DDF-3604](https://codice.atlassian.net/browse/DDF-3604)
#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
